bin/capsule-certs-generate

```diff
@@ -3,53 +3,75 @@ require 'rubygems'
 require 'kafo'
 require 'tempfile'
 
-def self.temp_file(name, content)
-  f = Tempfile.open([name, '.yaml'])
-  f.write content
-  f.close
-  f
+class CapsuleCertsGenerate
+  INSTALLER_DIR = '.'
+
+  attr_accessor :answers_file, :config_file
+
+  def generate_answers_file
+    temp_file('capsule-certs-answer',
+              <<-EOF
+                certs:
+                  generate: true
+                  deploy: false
+                  group: foreman
+                  org: Default_Organization
+                capsule_certs: true
+              EOF
+             )
+  end
+
+  # rubocop: disable Metrics/MethodLength
+  def generate_config_file(ans_f)
```

Comment:
Nitpick - readable (even if longer) variables help the reviewer and readers later of code to spend less time interpreting (e.g. ans_f vs. answer_file).